doc/lua_api.txt

```diff
@@ -3211,7 +3211,10 @@ This is basically a reference to a C++ `ServerActiveObject`
       }
 
 ##### LuaEntitySAO-only (no-op for other objects)
-* `set_velocity({x=num, y=num, z=num})`
+* `set_velocity({x=num, y=num, z=num}, properties)`
```

Comment:
I think it would probably be better to do `add_velocity` as a separate method to `set_velocity`.

However, the docs for this would be better as:

```
* `set_velocity(pos, options)`
    * pos - a position, `{ x=num, y=num, z=num }`
    * An optional table
        * `relative` - set to true to change velocity relatively. Defaults to false.
```

Reply:
done